client/src/components/SystemStatus/SystemStatus.module.scss

```diff
@@ -0,0 +1,42 @@
+.content {
+  /* WARNING: Mimicked on: History, Allocation, DataFiles, DataFilesProjectsList, DataFilesProjectFileListing, PublicData */
+  padding-top: 1.75rem; /* ~28px (22.5px * design * 1.2 design-to-app ratio) */
+  padding-left: 1.5em; /* ~24px (20px * design * 1.2 design-to-app ratio) */
+
+  /* As a flex child */
+  flex-grow: 1;
+  height: 85%;
+  width: 100% !important;
+}
+
+.sysmon-content {
+  /* WARNING: Mimicked on: History, Allocation, DataFiles, DataFilesProjectsList, DataFilesProjectFileListing, PublicData */
+  padding-top: 1.75rem; /* ~28px (22.5px * design * 1.2 design-to-app ratio) */
+  padding-left: 1.5em; /* ~24px (20px * design * 1.2 design-to-app ratio) */
+
+  /* As a flex child */
+  flex-grow: 1;
+  height: 15%;
+  width: 100% !important;
+}
+
+.section {
+  width: 50% !important;
```

Comment:
**UI Problem**
This seems to be the cause of a very narrow table on a narrow screen.

<details><summary>Screenshot</summary>

![core-portal 807 screenshot 50%](https://github.com/TACC/Core-Portal/assets/62723358/bb0980d0-e32f-4637-a53d-1acecaa89c0b)

</details>

A `min-width` should prevent the table from becoming too narrow, e.g.

```suggestion
.section {
  width: 50% !important;
  min-width: 500px;
}
```

**Alternative**

Try out `twoColumn` layout at [`contentLayoutName`](https://github.com/TACC/Core-Portal/pull/807/files#diff-8147f2d068a25c5cd46955dcee102197a77e0001d770c53c95b92484492733e4R55).

Reply:
I am trying this. It does not work immediately. I investigate.

Reply:
I succeeded in #816.